packages/@lwc/integration-karma/test/light-dom/slot-fowarding/slots/reactivity/x/lightContainer/lightContainer.html

```diff
@@ -0,0 +1,19 @@
+<template lwc:render-mode="light">
+    <x-light-light-slot data-id="lightLight">
+        <p slot={lightLightUpper}>Upper slot content</p>
+        <p slot={lightLightLower}>Lower slot content</p>
+        <p>Default slot content</p>
+    </x-light-light-slot>
+
+    <x-light-shadow-slot data-id="lightShadow">
+        <p slot={lightShadowUpper}>Upper slot content</p>
```

Comment:
Add if:true={} tests here because this will use the static diffing algo. Validate it doesn't pose any problems

Reply:
added in 6896343577b6576c9ab1b527d9070b8c80ba91ec